src/renderer/components/TimelineSpace/Contents/Search/Toots.vue

```diff
@@ -0,0 +1,28 @@
+<template>
```

Comment:
__[stylelint]__ <sub>posted by Sider</sub>

Unexpected empty source (no-empty-source)

Reply:
This component does not need any extra SCSS.